stock_orderpoint_mto_as_mts/models/product_product.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_warehouse_missing_orderpoint_for_mto(self):
+        res = {}
+        wh_obj = self.env["stock.warehouse"]
+        for product in self:
```

Comment:
```suggestion
        for product in self:
            if not product.purchase_ok:
                continue
```
There's no point to create a orderpoints for products that can not be purchased.

That is in conflict with the validation done in `purchase_reorder_control`
_Unpurchaseable Products cannot be reordered_
https://github.com/OCA/purchase-workflow/blob/e113f0a3a046ad832f2e324df50301c276932fa4/purchase_reorder_control/models/stock_warehouse_orderpoint.py#L21